lib/chefdepartie/cookbook.rb

```diff
@@ -28,6 +33,7 @@ def self.upload_cookbooks(path, books)
 
       books = books.collect do |name|
         next if Cache.cache(File.join(path, name))
+        next if File.file?(File.join(path, name))
```

Comment:
I guess this is because it might erroneously try to upload a single file as a cookbook? Should we make this 'next unless File.dir?'?